docs/tools/datadoc.md

```diff
@@ -382,15 +382,15 @@ The following formats are currently available:
     This will show the same output as in Ex 4.
 
 
-### load
-Loads documented dataset from a storage.
-Running `datadoc load --help` will show the following help message:
+### fetch
+Fetch documented dataset from a storage.
```

Comment:
Does it fetch the documented dataset or fetch " the documentation of the dataset(s) available in the triplestore"?

Reply:
It fetch the actual dataset.